gateway/header.go

```diff
@@ -12,6 +12,33 @@ import (
 	"github.com/grpc-ecosystem/grpc-gateway/runtime"
 )
 
+const XForwardedFor = "X-Forwarded-For"
+
+// GetGeoHeaders returns a slice of x-geo- headers.
+func GetGeoHeaders() []string {
+	return []string{
+		"x-geo-org",
+		"x-geo-country-code",
+		"x-geo-country-name",
+		"x-geo-region-code",
+		"x-geo-region-name",
+		"x-geo-city-name",
+		"x-geo-postal-code",
+		"x-geo-latitude",
+		"x-geo-longitude",
+	}
+}
```

Comment:
Why is this a func? If it's meant to make it extendable, a variable would just as well

```suggestion
var GetGeoHeaders = []string{
		"x-geo-org",
		"x-geo-country-code",
		"x-geo-country-name",
		"x-geo-region-code",
		"x-geo-region-name",
		"x-geo-city-name",
		"x-geo-postal-code",
		"x-geo-latitude",
		"x-geo-longitude",
}
```

Reply:
The reason why I made it as a func is assuming that this headers are meant to be constant. Such approach ensures that there is no way we can get mutated value.